.env.sample

```diff
@@ -8,14 +8,14 @@ REACT_APP_HELX_SEARCH_URL='https:\/\/heal-dev.apps.renci.org'
 # Should be a project token/GA property. Make sure to configure details
 # in env.json
 REACT_APP_ANALYTICS=''
-# Enables the TranQL tab in search concept popups
-REACT_APP_TRANQL_ENABLED='true'
 # Points the UI to TranQL. TranQL often runs on /tranql
 REACT_APP_TRANQL_URL='https:\/\/heal-dev.apps.renci.org\/tranql\/'
 # Brand setting. <braini, catalyst, heal, scidas, eduhelx, helx>
 REACT_APP_UI_BRAND_NAME='catalyst'
 # Hide support page sections if necessary. <'community', 'documentation'>. Comma-delimited string, e.g. "community,documentation"
 REACT_APP_HIDDEN_SUPPORT_SECTIONS=''
+# Hide tabs in the result modal/cards. <'overview', 'studies', 'cdes', 'kgs', 'tranql', 'robokop'>. Comma-delimited string, e.g. "tranql,robokop"
```

Comment:
I would add a note here to clarify that this only removes these tabs from the modal, not from the cards on the main concept search page (as far as I can tell -- please correct me if I'm wrong!).